### ISSUE TITLE
Can't upload >100gb file

### ISSUE THREAD

**Issue by DJSab** [TTLabs/EvaporateJS#432]
I'm trying to upload a 181gb file but after 2hrs of waiting it was 40% done, I got an error 400 on PUT request. I don't know what's causing it.

**Comment by jakubzitny** [comment]
Is this the same as #431? Did it happen more than once? Are you sure the the s3 configuration is correct?

**Comment by DJSab** [comment]
yes it is correct, I even uploaded 80gb file successfully with the same config.

**Comment by DJSab** [comment]
I just noticed the page dies after an hour while still uploading.  @jakubzitny any idea why? thank you.
![image](https://user-images.githubusercontent.com/22069587/59968889-4c475900-9574-11e9-9d9b-f0d19862ecce.png)

**Comment by jakubzitny** [comment]
Well, there is not much we can do without a code example. Are you using your own implementation with EvaporateJS or just the examples from the repo?
